deployment/arm/cosmosdb.json

```diff
@@ -0,0 +1,85 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
```

Comment:
does this need to be in its own file? Cant we merge it with the existing JSON files?